pkg/etl/location/location.go

```diff
@@ -0,0 +1,184 @@
+package location
```

Comment:
how did the *.sqlite3 files get generated? are there instructions for keeping those up to date?